composer.json

```
@@ -14,7 +14,9 @@
   "minimum-stability": "dev",
   "require": {
     "pimcore/compatibility-bridge-v10": "^1.0",
-    "pimcore/pimcore": "^10.6 || ^11.0"
+    "pimcore/admin-ui-classic-bundle": "^1.6",
```

```suggestion
    "pimcore/admin-ui-classic-bundle": "^1.0",
```
same as in other PRs, there is no hard requirement to a specific version.

Also please remove the entry from the "suggest" section

---

Also this doc paragraph
https://github.com/pimcore/perspective-editor/blob/3114b9dcd206d1c5f0e25cd5169a9aa5a7a9da3e/doc/01_Installation.md?plain=1#L30-L37

---

and if possible also https://github.com/pimcore/perspective-editor/blob/3114b9dcd206d1c5f0e25cd5169a9aa5a7a9da3e/src/Resources/public/js/pimcore/perspective/events.js#L46-L53